prax.c

```diff
@@ -179,20 +179,20 @@ static int recv_nl_req(int conn, struct taskmsg *msg)
 
 static void *make_nl_req(int req, profile_t *process)
 {
-    struct taskmsg msg;
-    memset(&msg, 0, sizeof msg);
+    struct taskmsg *msg = calloc(1, sizeof *msg);
```

Comment:
I could have left this without the allocation and used passed the address-of just the same while it was being used in scope.  The real issue was the `task_ret` allocation.